test/grpc_test.rb

```diff
@@ -53,8 +54,11 @@ def test_errors_are_tagged_with_the_resource_identifier
 
   def test_rpc_server
     run_services_on_server(@server, services: [EchoService]) do
-      GRPC::ActiveCall.any_instance.expects(:request_response)
+      GRPC::ActiveCall.any_instance.expects(:request_response).twice
       @stub.an_rpc(EchoMsg.new)
+      
+      operation = @stub_return_op.an_rpc(EchoMsg.new)
+      operation.execute
```

Comment:
There are two comments and suggestions:

1. I would prefer to separate unit test for each scenarios.

2. Move `stub` initialization from `setup` to test method.
It would avoid reling tests on global state and show all dependencies in same test.
That improve readability of the test.

```ruby
def test_rpc_server_with_operation
  subject = build_insecure_stub(EchoStubReturnOp)
  GRPC::ActiveCall.any_instance.expects(:request_response)

  run_services_on_server(@server, services: [EchoService]) do
    operation = subject.an_rpc(EchoMsg.new)
    operation.execute
  end
end
```